lua/starfall/libs_cl/vgui.lua

```diff
@@ -0,0 +1,184 @@
+--- VGUI library. Used to create derma interfaces.
+-- @client
+local VGUI, _ = SF.Libraries.Register("vgui")
```

Comment:
I think it should be called vgui_library to keep same style as rest of libraries.